README.md

```diff
@@ -48,8 +48,20 @@ shut down immediately. They leave no traces (outside of `/nix/store`) on the hos
   NixOps can be used to deploy to various other backends like cloud providers.\
   Requires: [Nix](https://nixos.org/nix/), [VirtualBox](https://www.virtualbox.org)
 
+- [`./deploy-container-minimal.sh`](examples/deploy-container-minimal.sh) creates a
+  container defined by [minimal-configuration.nix](examples/minimal-configuration.nix) that
+  doesn't use the [secure-node.nix](modules/presets/secure-node.nix) preset.
+  Also shows how to use nix-bitcoin in an existing NixOS config.\
```

Comment:
In what way exactly does this show how to use nix-bitcoin in an existing NixOS config?

Reply:
Actually, there's not much to show. I've [extended a comment](https://github.com/fort-nix/nix-bitcoin/compare/c1df4cb37a79ef6480e099a661dffac87d3bde3f..5d3221845b0cccf31fafdfffb72aa9bae6d63770) to make it a bit clearer.

Reply:
Oh I see okay